packages/framework/aqueduct/src/container-runtime-factories/baseContainerRuntimeFactory.ts

```diff
@@ -61,6 +62,10 @@ export interface BaseContainerRuntimeFactoryProps {
 	 * created with this factory
 	 */
 	provideEntryPoint: (runtime: IContainerRuntime) => Promise<FluidObject>;
+	/**
```

Comment:
I would suggest linking to this property in container runtime which has a detailed comment.

Reply:
Same in other places, if any.